packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -97,4 +110,17 @@ export class NgxsStoragePlugin implements NgxsPlugin {
       })
     );
   }
+
+  private executeSerializationStrategies(
+    key: string,
+    action: (strategy: NgxsStoragePluginSerializationOptions) => void
```

Comment:
maybe extract to new type

```ts
type SerializeAction = (strategy: NgxsStoragePluginSerializationOptions) => void;

private executeSerializationStrategies(key: string, action: SerializeAction): void {

}
```

Reply:
@splincode I don't think it is necessary to extract this because it is found in only one place (in this declaration) and it is internal.